content/blog/why-switch-to-pulumi/index.md

```diff
@@ -0,0 +1,470 @@
+semantic understanding of the cloud to unlock greater insights and
+controls over managing cloud infrastructure. Pulumi leverages the
+familiar GPT experience you know, love, and use daily, so
+you can find and take action on any resource in your cloud
+environments.
+
+### Ok, but why does this matter enough for me to switch?
+
+Only you can answer that question, but there are some compelling reasons
+to adopt the platform trusted by hundreds of thousands of developers.
+
+**Faster Time to Value.** Your company moved to the cloud to
+increase innovation and reduce costs. However, getting infrastructure to
+developers is a frequent bottleneck for you which can slow down
+prototyping new products or shipping new features. Spinning up new
+development or testing environments takes days and rolling out
+production updates across many regions can take weeks. The existing
+tools don't allow you to set security and compliance guardrails and
+enable easy self-service of infrastructure.\
+\
+*Pulumi will speed up your deployments and time to value.*
+
+**Open Platform Commitment.** Some platforms, such as Terraform, have
+altered their licensing and introduced uncertainty. They are no longer
+true open source, and they tie their previously open source software to
+their commercial services. The lack of an open source approach fragments
+the community and introduces proprietary constraints to cloud
+infrastructure.\
+\
+*Pulumi provides you with an approach to open source and community that
+provides stability and choice.*
+
+**Increased Productivity:** You expect your tools
+and workflows to keep up with the industry: AI copilots, Intellisense,
+linting tools, testing frameworks, and CICD pipelines. But most of these
+innovations are just for building applications and services and not for
+infrastructure, configurations, policies, and secrets. Existing
+infrastructure tools are fraught with bad UX, rigid scripting languages,
+and error prone manual processes.
+
+*Pulumi will increase your productivity and velocity through better tooling.*
+
+**Infinite Scale.** Modern cloud architectures are distributed systems
+that are microservices that are dynamic and ephemeral in nature. The
+number of infrastructure resource types and the configurable input
+properties is staggering. You face the daunting task of how to
+combine these resources to solve their unique problems. Without software
+engineering, managing these modern distributed systems is fruitlessly
+manual.
+
+*Pulumi uses software engineering to tackle the scale of infinite
+combinations.*
+
+{{% notes type="info" %}}
+Customers of Pulumi frequently experience 100% productivity increases, 99% time saved, and 10x acceleration when using Pulumi compared to what they were using before.
+
+![modern-benefits](modern-benefits.png)
+{{% /notes %}}
+
+## Frequently Asked Questions - Why Pulumi?
+
+Some frequently asked questions about why Pulumi based on use cases and
```

Comment:
suggestion: Here are some frequently asked questions about why you should choose Pulumi based on the following use cases and alternatives.